src/main/ansible/vagrant.yml

```diff
@@ -62,10 +62,12 @@
 
   - name: Copy test-bags to server
     copy:
-      src: "../../test/resources/vault/stores/pdbs/bags/"
+      src: "../../test/resources/vault/stores/pdbs/bags/9da0541a-d2c8-432e-8129-979a9830b427"
       dest: "{{ stores_dir }}/pdbs/ab/123456789012345678901234567890"
       mode: "0775"
       directory_mode: "0775"
+    # TODO skip all git-ignored files, not just hidden files in the bags folder
+    # workaround for now: just the one bag we have at the moment
```

Comment:
Tried the variant below. It should skip hidden files (like `.DS_Store`), but just at the bag level. Didn't get it working.

```
      src: "{{ item }}"
       dest: "{{ stores_dir }}/pdbs/ab/123456789012345678901234567890"
       dest: "{{ stores_dir }}/pdbs/ab/123456789012345678901234567890"
       mode: "0775"
       mode: "0775"
       directory_mode: "0775"
       directory_mode: "0775"
  with_fileglob: "../../test/resources/vault/stores/pdbs/bags/*"
```

Reply:
Tried http://docs.ansible.com/ansible/latest/find_module.html 
but it finds files on the guest while I want files on the host. Maybe the same problem in the previous message.

Reply:
Tried various variants I found via [stack overflow](https://stackoverflow.com/questions/18900236/run-command-on-the-ansible-host). The one below (and others) return a MODULE FAILURE with  `sudo: a password is required`. Anyone any clue?

    find:
      paths: "../../test/resources/vault/stores/pdbs/bags"
      recurse: "yes"
    delegate_to: 127.0.0.1
    register: bag_files

Reply:
I'd say, don't spend too much time on this! It's just test data, right? Just delete these files via IntelliJ or the command line and focus on the important things first!

Reply:
maybe remove this comment and follow my last comment?
Please don't spend too much time on trying to fix this, though!!!